index.html

```diff
@@ -718,9 +721,10 @@ <h2>Queue the PerformanceObserver task</h2>
             <var>relevantGlobal</var>'s <a>list of registered performance
             observer objects</a>.
             </li>
-            <li>For each <a>PerformanceObserver</a> object <var>po</var> in
+            <li>For each <a>PerformanceObserver</a> object <var>registeredObserver</var> in
```

Comment:
`notifyList` is a list of "registered performance observer", not of PerformanceObserver

Reply:
Yes, missed that fix, thanks